scripts/deploy.ts

```diff
@@ -59,19 +62,28 @@ async function main() {
     createStatement
   );
   let receipt = await tx.wait();
-  const [, createEvent] = receipt.events ?? [];
+  const [, createEvent] = receipt?.logs.filter(isEventLog) ?? [];
   const tableId = createEvent.args!.tableId;
   console.log("Healthbot table created as:", `healthbot_${chainId}_${tableId}`);
 
   // Insert first row into health bot table
   const runStatement = `insert into healthbot_${chainId}_${tableId} values (1);`;
+  // Add 20% buffer to gas limit to avoid out-of-gas errors
```

Comment:
txs were failing when i tried to deploy to a live network, i.e., sepolia. the gas limit was hitting 99%+. this adds a buffer and fixed the issue.